apps/angular/pipe-easy/src/app/app.component.ts

```diff
@@ -1,21 +1,18 @@
-import { NgFor } from '@angular/common';
 import { Component } from '@angular/core';
+import { IndexedPipe } from './indexed.pipe';
 
 @Component({
   standalone: true,
-  imports: [NgFor],
+  imports: [IndexedPipe],
   selector: 'app-root',
   template: `
-    <div *ngFor="let person of persons; let index = index">
-      {{ heavyComputation(person, index) }}
-    </div>
+    @for (person of persons; track person; let index = $index) {
```

Comment:
you don't need to use an alias, you can use $index directly inside your for block

alias are good if you have mutliple nester for blocks

Reply:
Good to know. Thanks